Tasks/UsePythonVersionV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,6 +6,8 @@
   "loc.group.displayName.advanced": "Advanced",
   "loc.input.label.versionSpec": "Version spec",
   "loc.input.help.versionSpec": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)",
+  "loc.input.label.allowUnstable": "Allow downloading unstable releases",
+  "loc.input.help.allowUnstable": "Allow downloading unstable python versions from registry.",
```

Comment:
What is a registry here? Could it make sense to describe it more everywhere in messages (you can maybe use description from here https://github.com/actions/python-versions) and add the link to this?

Reply:
Explained it and added links to both python-versions registry and the manifest in all messages: https://github.com/microsoft/azure-pipelines-tasks/pull/15820/commits/e4494dea0f8461e2120d0aaaa18c180d334e424d